beancount/parser/parser.c

```diff
@@ -15,189 +15,248 @@ extern YY_DECL;
 
 extern const char* getTokenName(int token);
 
-/* TODO(blais,dnicolodi): Convert those to locals eventually. */
-yyscan_t _scanner;
+/* Placeolder object for missing cost specifications. */
+PyObject* missing_obj;
 
-/* The current builder during parsing (as a global variable for now). */
-PyObject* builder = 0;
+typedef struct {
+    PyObject_HEAD
+    yyscan_t scanner;
+    PyObject* builder;
+} Parser;
 
-/* A reference to a Python-defined constant object used as a placeholder for
-   missing cost specifications. */
-PyObject* missing_obj = 0;
+PyDoc_STRVAR(parser_doc, "");
```

Comment:
Why no docstring?

Reply:
Because I am lazy and I forgot to fill them in before submitting the PR. I'll add them.